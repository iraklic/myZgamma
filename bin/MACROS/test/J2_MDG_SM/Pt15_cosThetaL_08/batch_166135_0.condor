000 (166135.000.000) 11/04 11:29:49 Job submitted from host: <131.225.191.163:50304>
...
001 (166135.000.000) 11/04 11:30:17 Job executing on host: <131.225.205.216:52397>
...
006 (166135.000.000) 11/04 11:30:25 Image size of job updated: 78016
...
006 (166135.000.000) 11/04 11:31:25 Image size of job updated: 81240
...
005 (166135.000.000) 11/04 12:04:42 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:34:06, Sys 0 00:00:03  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:34:06, Sys 0 00:00:03  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	53265  -  Run Bytes Sent By Job
	3008743  -  Run Bytes Received By Job
	53265  -  Total Bytes Sent By Job
	3008743  -  Total Bytes Received By Job
...
