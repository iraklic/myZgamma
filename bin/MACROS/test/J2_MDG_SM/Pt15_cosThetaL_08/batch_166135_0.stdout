
CMSSW on Condor

started at Fri Nov  4 11:30:17 CDT 2011

parameter set:
CONDOR_CLUSTER: 166135
CONDOR_PROCESS: 0
CMSSW_DIR: /uscms_data/d2/iraklich/Zgamma/CMSSW_4_2_0
RUN_DIR: /uscms_data/d2/iraklich/Zgamma/CMSSW_3_8_4_1/src/ElectroWeakAnalysis/MultiBosons/bin/MACROS/test/myMinexam
PARAMETER_SET: output_Mu.txt
NUM_EVENTS_PER_JOB: 

Writing final parameter-set: batch_166135_0.py to RUN_DIR: /uscms_data/d2/iraklich/Zgamma/CMSSW_3_8_4_1/src/ElectroWeakAnalysis/MultiBosons/bin/MACROS/test/myMinexam

run: myMinexam > batch_166135_0.log 2>&1\n

finished at Fri Nov  4 12:04:42 CDT 2011
